packages/visualizations/addon/templates/components/navi-visualizations/table.hbs

```diff
@@ -5,36 +5,48 @@
 </div>
 <div class="table-widget__horizontal-scroll-container">
   <div class= 'table-widget-container'>
-      {{#sortable-group class='table-header-row' direction='x' onChange='updateColumnOrder' as | group |}}
-        {{#each columns as |column|}}
+      {{#if isEditing}}
+        <div class='table-header-row'>
+          {{#each columns as |column|}}
+            <div class='table-header-cell {{column.type}}'>
+              {{input
+                class='table-header-cell__input'
+                value=(readonly column.displayName)
+                focus-out=(action 'updateHeaderDisplayName' column)
+              }}
+            </div>
```

Comment:
Is there a way that we can add error state/error message?

Reply:
I feel that we may not need an error message here. Is there anything we need to validate here?